### PR TITLE
[INFRA] Integrate feedback of the first release

### PR DESCRIPTION
**Pull request by tbouffard**
Integrate feedback from
- 1st manual package generation
- 1st submission rejection (https://github.com/process-analytics/bpmn-visualization-R/issues/10#issuecomment-913705549)
  - description title: use title case
  - do not package the README.md file: it generates warnings because of unreachable links (contributing.md not in the package)
  - do not package the `doc` folder: only needed for the README.md file, non standard folder
- the github worklow now builds the source package and run checks on it as done during the CRAN submission to detect errors earlier

Also read: https://jef.works/blog/2018/06/18/get-your-package-on-cran-in-10-steps/

Covers #10

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -149,14 +149,17 @@ Create a new PR and merge it, if the version in the [DESCRIPTION](./DESCRIPTION)
...
-- Go to the parent folder of the project. You should find a file named like `bpmnVisualization_X.Y.Z.tgz`.
-- Fill all the fields of this [web form](https://xmpalantir.wu.ac.at/cransubmit/), and load `bpmnVisualization_X.Y.Z.tgz` file.
-- Submit
-<p align="center"><img title="Submitting to CRAN" src="./doc/cran-submitting.png" alt="Submitting to CRAN"></p>
+- Build the source package. Please read [the CRAN Submission policies first](https://cran.r-project.org/web/packages/policies.html#Submission)
+  - Solution 1: Build the source package with RStudio 
+    - Open the project in RStudio
+    - On the **Build** tab, click on **More** > **Build Source Package**
+    - Go to the parent folder of the project. You should find a file named like `bpmnVisualization_X.Y.Z.tgz`.
+  - Solution 2. **TODO more details**
```
Comment (csouchet) [inline CONTRIBUTING.md:157]: Is it finish ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -149,14 +149,17 @@ Create a new PR and merge it, if the version in the [DESCRIPTION](./DESCRIPTION)
...
+    - from the project directory, run `R CMD build .` **TODO this may require more command to run first**
```
Comment (csouchet) [inline CONTRIBUTING.md:158]: Same ?

File: CONTRIBUTING.md (hunk 1 above)
Comment (tbouffard) [reply CONTRIBUTING.md:157]: I was waiting for more feedback from the CRAN submission. This is what is currently implemented in the GitHub workflow.
I can remove the TODO and if the current procedure is not ok, we can create a new PR to update the documentation.
Is that ok?

Comment (csouchet) [reply CONTRIBUTING.md:157]: No problem 👍🏼

**Review by csouchet — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (tbouffard) [reply CONTRIBUTING.md]: done in 71edea5

**Pull request merged**